internal/pkg/externalplugins/plugins_test.go

```diff
@@ -67,10 +65,6 @@ func TestStartLoadConfig(t *testing.T) {
 		t.Errorf("Different TiCommunityLgtm len: Got \"%d\" expected \"%d\"",
 			len(config.TiCommunityLgtm), expectLen)
 	}
-	if config.TiCommunityLgtm[expectLen-1].ReviewActsAsLgtm != true {
```

Comment:
Removing this configuration causes the test to be invalid. (because there is no configuration update at all) So we should use another plugin configuration (with other configuration items) to verify that the polling update configuration mechanism is working.

Reply:
Make sense. Thanks for your review!

I've pushed a commit 2eae1127eb5e4322d4694d38e380a60b1e6c103d. Hopefully I understand it correctly.

Reply:
I think I miss the validation part.

Reply:
pushed